### PR TITLE
[Ragile] Fan control hld

### PR DESCRIPTION
**Pull request by pettershao-ragilenetworks**
As below pr mentioned by jleveque, need to upload Ragile fan control policy, please help check! @jleveque @sujinmkang @yxieca
https://github.com/Azure/sonic-buildimage/pull/7620

### CODE REVIEW

File: doc/ragile fancontrol hld.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+## Table of Content 
+
+[TOC]
+
+### Revision  
+
+| Rev  | Date | Author | Change Description |
+| ---- | ---- | ------ | ------------------ |
+|      |      |        |                    |
```
Comment (sujinmkang) [inline doc/ragile fancontrol hld.md:11]: Can we have the information here?

File: doc/ragile fancontrol hld.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
+# Ragile Fan control HLD #
+
+## Table of Content 
+
+[TOC]
```
Comment (sujinmkang) [inline doc/ragile fancontrol hld.md:5]: Also update this.

**Comment by pettershao-ragilenetworks** [comment]
@sujinmkang done, help check!

**Review by sujinmkang — APPROVED** [review]

**Comment by sujinmkang** [comment]
@pettershao-ragilenetworks Can you please update the description with the information of the enhancement in this HLD?

**Pull request merged**